doc/modules/ROOT/pages/ops.adoc

```diff
@@ -84,8 +84,7 @@ Returns::
 
 === `:interrupt`
 
-Attempts to interrupt some code evaluation. When interruption succeeds, the thread used by for
-code evaluation changes, while Clojure dynamic bindings are preserved, other `ThreadLocals` are not. So when running code intimately tied to the current thread identity, best avoid interruptions.
+Attempts to interrupt some code evaluation.
```

Comment:
Why was this changed?

Reply:
This must have happened when I ran `lein docs` following along the https://nrepl.org/nrepl/hacking_on_nrepl.html guide. My suspicion is that someone had modified the docs in an earlier commit but not the function docstring used to generate this file. I'll add the more detailed description to the function so this doesn't happen again. Good catch though!